feedback/views.py

```diff
@@ -32,3 +34,10 @@ def get(self, request, format=None):
         feedbacks = Feedback.objects.all()
         serializer = FeedbackSerializer(feedbacks, many=True)
         return Response(serializer.data)
+
+
+@method_decorator(xframe_options_exempt, name='dispatch')
+class FeedbackCreate(CreateView):
+    form_class = FeedbackForm
+    model = Feedback
+    success_url = "/thankyou"
```

Comment:
Shouldn't this be configured using `reverse()`?

Reply:
True. Let's fix that.

Reply:
Done also.